common.ts

```diff
@@ -70,6 +78,22 @@ export class Config {
     const chainid = process.env.CHAIN_ID || '420'
     return parseInt(chainid, 10)
   }
+
+  public static L1UserPrivateKey(): string {
+    return process.env.L1_USER_PRIVATE_KEY
+  }
+
+  public static L2UserPrivateKey(): string {
+    return process.env.L2_USER_PRIVATE_KEY
+  }
+
+  public static L1MessengerAddress(): string {
```

Comment:
We should use the address manager to get the `L1_MESSENGER_ADDRESS`

Reply:
This pattern would be helpful for removing `Config.StateCommitmentChainContractAddress` as well

Reply:
You mean pull the address from the address manager contract?

Reply:
Made this change!